integrations/nielsen-dcr/lib/index.js

```diff
@@ -223,20 +224,25 @@ NielsenDCR.prototype.getContentMetadata = function(track, type) {
 };
 
 /**
- * Get ad content metadata from track event
+ * Get ad metadata from track event
  *
  * @api private
  */
 
 NielsenDCR.prototype.getAdMetadata = function(track) {
   var type = track.proxy('properties.type');
+
   if (typeof type === 'string') type = type.replace('-', '');
 
+  var customAssetId = track.proxy(
+    'properties.' + this.options.adAssetIdPropertyName
```

Comment:
What happens if this is just an empty string?

Reply:
Good catch - def need to fix this. `track.proxy` would return the full properties object in this case.